debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -2467,26 +2467,6 @@ public void shouldProduceMessagesOnlyForConfiguredTables() throws Exception {
     }
 
     @Test
-    @FixFor("DBZ-2885")
-    @SkipWhenDecoderPluginNameIsNot(value = SkipWhenDecoderPluginNameIsNot.DecoderPluginName.PGOUTPUT, reason = "Publication configuration only valid for PGOUTPUT decoder")
-    public void shouldThrowWhenTableFiltersIsEmpty() throws Exception {
-        final LogInterceptor logInterceptor = new LogInterceptor();
-
-        TestHelper.dropAllSchemas();
-        TestHelper.dropPublication("cdc");
-        TestHelper.executeDDL("postgres_create_tables.ddl");
-        TestHelper.execute(SETUP_TABLES_STMT);
-
-        Configuration.Builder configBuilder = TestHelper.defaultConfig()
-                .with(PostgresConnectorConfig.PUBLICATION_NAME, "cdc")
-                .with(PostgresConnectorConfig.PUBLICATION_AUTOCREATE_MODE, PostgresConnectorConfig.AutoCreateMode.FILTERED.getValue())
-                .with(PostgresConnectorConfig.TABLE_INCLUDE_LIST, "nonexistent.table");
-
-        start(PostgresConnector.class, configBuilder.build());
-        assertConnectorNotRunning();
-        assertTrue(logInterceptor.containsStacktraceElement("No table filters found for filtered publication cdc"));
-    }
-
```

Comment:
How is this test removal related?

Reply:
I think it's related to some merge conflicts that I have. I will fix it.